Paintroid/src/main/java/org/catrobat/paintroid/ui/MainActivityNavigator.java

```diff
@@ -135,6 +137,19 @@ public void startWelcomeActivity(@ActivityRequestCode int requestCode) {
 		mainActivity.startActivityForResult(intent, requestCode);
 	}
 
+	@Override
+	public void startShareImageActivity(Bitmap bitmap) {
+		Uri uri = FileIO.saveBitmapToCache(bitmap, mainActivity);
+		if (uri != null) {
+			Intent shareIntent = new Intent();
+			shareIntent.putExtra(Intent.EXTRA_STREAM, uri);
+			shareIntent.setDataAndType(uri, mainActivity.getContentResolver().getType(uri));
+			shareIntent.setFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
+			shareIntent.setAction(Intent.ACTION_SEND);
+			mainActivity.startActivity(Intent.createChooser(shareIntent, "Send image via"));
```

Comment:
don't use hardcoded strings that need to be translated. extract **"send image via"** to string resource